server.rb

```diff
@@ -143,7 +174,6 @@
 end
 
 # Get List of testers
-get '/testers' do
```

Comment:
Why was this kine removed?

Reply:
Removed by abuse when I fixed merge conflict, thanks for verifying!